src/pages/CustomerInvoicePage.js

```diff
@@ -133,6 +134,7 @@ export const CustomerInvoicePage = ({
     } = newDataTableStyles;
     return columns.map(({ key: colKey, type, width, alignText }, index) => {
       const isLastCell = index === columns.length - 1;
+      const isDisabled = isFinalised || (rowState && rowState.disabled);
```

Comment:
`rowState.isDisabled`?
Could use `rowData.transaction.isFinalised?

Reply:
hmm, is `rowState.isDisabled` ever even set anywhere? What would be the use case for having a particular row disabled? Would also be very difficult to set each `rowState` field to `disabled` when finalising. Maybe should just remove `rowState` from this condition? Can use `rowData.transaction.isFinalised`, dunno if there's much to gain, pretty much have to use `pageObject.isFinalised` in the dependencies anyway?

Reply:
Never mind, I assume this is actually separated for handling stuff the the customer invoice list where only some rows are disabled ?

Reply:
ahh that's the use case! I guess I'll keep it in then 🤣

Reply:
Well, maybe that'd make sense if we're going to factor out renderCells